### ISSUE TITLE
Current build not watching for changes?

### ISSUE THREAD

**Issue by gerald1248** [arnehilmann/markdeck#40]
I've probably misconfigured something at my end, but markdeck no longer rebuilds when I change slides.md or files in assets/? New repo, created today using markdeck pull and scaffold.

The presentation also shows the speaker notes by default - even for slides that don't have any.

Markdeck version: 0.5
Catilina 10.15.4

When I revert to version 0.34's start and stop scripts, these problems disappear.

**Comment by arnehilmann** [comment]
That is really strange: After removing all locally built markdeck images, I downloaded the latest official images via `./markdeck start`. Changes of `slides.md` or in `assets/*` all trigger a rebuild... could you provide a minimal tarball of your non-working slide deck?!

**Comment by gerald1248** [comment]
Have tried again: fresh folder,

```$ curl -LO https://raw.githubusercontent.com/arnehilmann/markdeck/master/markdeck
$ chmod a+x markdeck
$ ./markdeck pull
$ ./markdeck scaffold
$ ./markdeck start
```

Then I added a slide to `slides.md` in another terminal - but the original markdeck terminal doesn't rebuild.

[issue40.zip](https://github.com/arnehilmann/markdeck/files/4715458/issue40.zip)

**Comment by arnehilmann** [comment]
*ouch* please try `mkdir themes` before running `./markdeck`...
that will be addressed in 0.51, which will be released in the next couple of days.

**Comment by RyanBErickson** [comment]
Ran into this tonight as well...  Thanks for the quick workaround (adding /themes directory solved it, of course)!

**Comment by gerald1248** [comment]
Same here, thank you!

**Comment by arnehilmann** [comment]
fixed with v0.51
